jelares/portfolio/target/portfolio-1/css/homeStyle.css

```diff
@@ -0,0 +1,57 @@
+* {
+    font-family: 'Open Sans', sans-serif;
+}
+
+h1, h2, h3, h4 {
+    font-family: 'Roboto', sans-serif;
+}
+
+.full-screen {
+    margin: 0;
+    height: 100vh;
+}
+
+.parallax {
+    /* Set rules to fill background */
+    min-height: 100%;
+        
+    /* Set up proportionate scaling */
+    width: 100%;
+    height: auto;
+
+    /* Create the parallax scrolling effect */
+    background-attachment: fixed;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+}
```

Comment:
Style encourages ordering declarations alphabetically:
https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order

Reply:
I will fix these as we go along